.github/workflows/ci.yml

```diff
@@ -21,10 +21,40 @@ jobs:
     steps:
       - name: Check out APISIX plugins
         uses: actions/checkout@v2.4.0
-        with:
-          path: apisix-plugin
 
       - name: Check out APISIX repo
         run: |
           sudo ./ci/utils/linux-common-runnner.sh get_apisix_code
-        working-directory: apisix-plugin
+
+      - name: Install custom module
+        run: |
+          sudo ./ci/utils/linux-common-runnner.sh install_module
+
+      - name: Launch APISIX common services
+        run: |
+          sudo make ci-env-up project_compose_ci=ci/pod/docker-compose.common.yml
+        working-directory: workbench
+
+#      - name: Launch APISIX deps env
+#        run: |
+#          sudo make ci-env-up
+#          ./ci/linux-ci-init-service.sh
+#        working-directory: workbench
+
+      - name: Run centos7 docker and mapping apisix into container
```

Comment:
I think we should run the test cases in Ubuntu.

Reply:
OK. I will fix it.